client/signup/steps/domains/style.scss

```diff
@@ -346,6 +341,7 @@ body.is-section-signup.is-white-signup {
 			margin: 0;
 
 			@include break-small {
+				width: 290px;
```

Comment:
The changes in this file set the sidebar to a static width of 290px for anything bigger than mobile.